web/src/i18n/translations/ja.yml

```
@@ -1,582 +1,585 @@
-Successfully created webhook!: Webhookの作成に成功しました。
-Failed to delete webhook.: Webhookの削除に失敗しました。
-Successfully deleted webhook!: Webhookの削除に成功しました。
-Failed to update webhook.: Webhookの更新に失敗しました。
-Successfully updated webhook!: Webhookの更新に成功しました。
-Failed to create integration.: インテグレーションの作成に失敗しました。
-Successfully created integration!: インテグレーションの作成に成功しました。
-Failed to create project.: プロジェクトの作成に失敗しました。
-Successfully created project!: プロジェクトの作成に成功しました。
-Failed to delete workspace.: ワークスペースの削除に失敗しました。
-Successfully deleted workspace!: ワークスペースの削除に成功しました。
+Copy: コピー
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)